static/js/publisher/publicise.js

```diff
@@ -149,7 +149,12 @@ function initEmbeddedCardPicker(options) {
   function updateCardSize() {
     // calulate frame height to be a bit bigger then content itself
     // to have some spare room for responsiveness
-    if (previewFrame.offsetParent) {
+    if (
+      previewFrame.offsetParent &&
+      previewFrame.contentWindow &&
+      previewFrame.document &&
+      previewFrame.document.body
```

Comment:
You just need to check the preview has a body to cover this error: `previewFrame.contentWindow.document.body`

Reply:
That's some good debugging! I obviously wasn't clicking around fast enough. Anyway, fixed.